onnx/backend/test/cpp/driver/test_driver.h

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <queue>
+#include <string>
+#include <cstdlib>
+#include <cstdio>
+
+#include "onnx/onnx_pb.h"
+#include "onnx/proto_utils.h"
+#include "onnx/onnxifi_loader.h"
+#include "onnxifi.h"
+
+#include "gtest/gtest.h"
+
+struct TestData{
```

Comment:
It would be good to add high level doxygen style comments on what the class and functions are about.